<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeMenu</class>
 <widget class="QWidget" name="HomeMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
    font-family: century gothic;
    font-size: 20px;
}

QToolButton
{
    background: #D1D2D2;
    border-radius: 30px;
}

QLabel
{
    color: black;
    border: 2px solid black
}

QPushButton
{
   border: 2px solid black;
   background-color: #D1D2D2;
   border-radius: 15px;
}

QPushButton:hover
{
   color: #D1D2D2;
   border-radius: 15px;
   background: #16A5C3;
}

QFrame
{
  background: #C7D1D1;
  border: 2px solid black
}
QWidget
{
   background-color:   #6dd5ed;
}
</string>
  </property>
  <widget class="QPushButton" name="reportButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
   border: 2px solid black;
   background-color: white;
   border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Reports</string>
   </property>
  </widget>
  <widget class="QPushButton" name="patientButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
   border: 2px solid black;
   background-color: white;
   border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Patient Records</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AppointmentButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
   border: 2px solid black;
   background-color: white;
   border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Appointments</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registrationButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
   border: 2px solid black;
   background-color: white;
   border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>New Registration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adminButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
   border: 2px solid black;
   background-color: white;
   border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Admin Only</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>381</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
  background: #C7D1D1;
  border: 2px solid black
}

QLabel
{
  background: white;
}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="companyLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton
{
   background-color: white;
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/kidneyIcon.png</normaloff>:/kidneyIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>800</width>
      <height>800</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Kidney Association of Belize</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="clock">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber
{
   background: white;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Today's Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="logOutButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome to the main menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
   border: 2px solid black;
   background-color: white;
   border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="patientRecButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
   border: 2px solid black;
   background-color: white;
   border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Patient Records</string>
   </property>
  </widget>
  <widget class="QPushButton" name="contactButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
   border: 2px solid black;
   background-color: white;
   border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Organization Contacts</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
